### PR TITLE
295 search imp

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
Improvements editing
#### Where should the reviewer start?
components/searchInput
#### How should this be manually tested?
http://localhost:8080
#### What are the relevant tickets?
#295 
#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/branch-295-search-imp

### CODE REVIEW

File: src/components/SearchInput/SearchInput.vue [hunk 1]
```diff
@@ -6,14 +6,26 @@
...
         name="search"
       />
     </div>
+    <div class="search-clear">
+      <svgicon
+        v-if="inputMode && value.length"
+        width="14"
+        height="14"
+        name="cancel"
+        @click="$emit('input', $event.target.value)"
```
Comment (lapinlapin) [inline src/components/SearchInput/SearchInput.vue]: @roma219 help me please in this moment.. this click not work, click.native too

Comment (roma219) [reply src/components/SearchInput/SearchInput.vue]: Well, $event.target  for this click event is icon, it doesn't have any value. You probably want to emit an empty string to clear the input.

**Review by roma219 — CHANGES_REQUESTED** [review]
The search input should not disapper when it is focused and mouse is moved away

File: src/components/SearchInput/SearchInput.vue [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
     </div>
+    <div
+      v-if="value.length"
+      class="search-clear"
+    >
+      <svgicon
+        width="14"
+        height="14"
+        name="cancel"
+        @click="$emit('input', ''); value = ''"
```
Comment (roma219) [inline src/components/SearchInput/SearchInput.vue:30]: image
?

**Review by roma219 — CHANGES_REQUESTED** [review]
image
Please remove word "Search" from placeholder

File: src/components/SearchInput/SearchInput.vue (hunk 2 above)
Comment (lapinlapin) [reply src/components/SearchInput/SearchInput.vue:30]: 1. emit - for a vuex search from str and update state
2. value = '' for a reset value in this input

**Review by roma219 — APPROVED** [review]

**Pull request merged**